config/routes.config.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express');
+const router = express.Router();
+
+const controller = require('../controllers/spotify.controller');
```

Comment:
```suggestion
const spotifyController = require('../controllers/spotify.controller');
```

Piensa que esto será escalable, es decir, tendremos muchos controladores, asi que si desde ya mismo os acostumbrais a ser descriptivos con las variables genial! fijate poniendo eso de una pasada sabrias que tiene esa variable, en cambio con tu nombre sabrias que es un controlador pero no de que.